src/site/topics/recursion/recursion.rst

```diff
@@ -381,22 +400,33 @@ Fibonacci
 
 * What is the computational complexity of ``recursiveFibonacci(n)``?
 * This may feel a little less straight forward compared to ``recursiveFactorial(n)``, but the idea is the same
+
     * The function has constant time operations
-    * But we see that there are recursive calls, so, how many times does this function get called?
+    * But there are recursive calls, so, how many times does this function get called?
 
-.. image:: fibonacci.png
-   :width: 500 px
-   :align: center
 
+.. figure:: fibonacci.png
+    :width: 500 px
+    :align: center
+
+    Visualization of the recursive function calls when calling ``recursiveFactorial(6)``. Unless it is the base case,
+    each call to ``recursiveFactorial`` produces two more recursive calls to ``recursiveFactorial``. Notice how many
+    times ``recursiveFactorial(2)`` is calculated --- 5.
```

Comment:
typos `Factorial`

Reply:
It took me far too long to see the issue.